features/search.feature

```diff
@@ -150,29 +159,59 @@ Scenario: search for engagement semesters only with semester
     Given I uncheck "Apps"
     And I uncheck "Organizations"
     And I uncheck "Users"
-    And I search for "fa"
-    # Then I should see "app 1"
-    # And I should see "app 2"
-    # Then I search for "spring"
-    # Then I should see "app 2"
-    # And I should see "app 3"
-
-Scenario: search for engagement semesters only with year
+    Then I search for "FALL 2015"
+    Then I should see "app 1"
+    And I should see "app 2"
+    And I should not see "app 3"
+    Then I search for "Spring 2016"
+    Then I should see "app 2"
+    And I should not see "app 1"
+    And I should not see "app 3"
+    
+Scenario: search for engagement semesters with all checked
+    Given I search for "FALL 2016"
+    Then I should see "app 1"
+    And I should not see "app 2"
+    And I should see "app 3"
+
+Scenario: search for engagement semesters with fuzzy input semester
+    Given I uncheck "Apps"
+    And I uncheck "Organizations"
+    And I uncheck "Users"
+    Then I search for "fa"
+    Then I should see "app 1"
+    And I should see "app 2"
+    And I should see "app 3"
+
+    Given I search for "spring"
+    Then I should not see "app 1"
+    And I should see "app 2"
+    And I should see "app 3"
+
+
+Scenario: search for engagement semesters with fuzzy input year
     Given I uncheck "Apps"
     And I uncheck "Organizations"
     And I uncheck "Users"
     And I search for "15"
-    # Then I should see "app 1"
-    # Then I search for "2013"
-    # Then I should see "app 3"
+    Then I should see "app 1"
+    And I search for "2015"
+    Then I should see "app 1"
+    Then I should see "app 2"
+    Then I should see "app 3"
 
-Scenario: search for engagement semesters only with semester and year
+    Given I search for "2013"
+    Then I should not see "app 1"
+    Then I should not see "app 2"
+    Then I should not see "app 3"
+
+Scenario: search for engagement semesters only with fuzzy input semester and year
     Given I uncheck "Apps"
     And I uncheck "Organizations"
     And I uncheck "Users"
-    And I search for "SPRING 2013"
-    # Then I should see "app 3"
-    # And I search for "Fa15"
-    # Then I should see "app 1"
+    And I search for "Fa15"
+    Then I should see "app 1"
+    Then I should see "app 2"
+    Then I should not see "app 3"
     
 
```

Comment:
Great tests but they don't seem to be relevant for this particular feature. Perhaps, you can make a new feature branch for this or continue to work on the branch for this feature.